source/locales.tex

```diff
@@ -4689,9 +4689,9 @@
     class InputIterator = istreambuf_iterator<charT> >
   class money_get : public locale::facet {
   public:
-    typedef charT               char_type;
-    typedef InputIterator       iter_type;
-    typedef basic_string<charT> string_type;
+    using   char_type = charT;
```

Comment:
It is a bit cute. I largely threw it in to see the reaction, as we work out how to align the new syntax.  I like it, but suspected it might not be the preferred choice.  Unless I hear support, I will left-align any remaining right-aligned identifiers, but retain the alignment on the '='.